source/Engine.cpp

```diff
@@ -1853,7 +1853,7 @@ void Engine::SpawnPersons()
 	if(!sum)
 		return;
 
-	// Although an attempt to spawn a person is made every 10 minutes on average,
+	// Although an attempt to spawn a person is made every 20 minutes on average,
```

Comment:
maybe just dont mention a specific value here, in case of further changes

Reply:
Any preference on rewording it, or just remove the phrase altogether?